apiserver/params/params.go

```diff
@@ -384,6 +384,7 @@ type AddApplicationUnits struct {
 	ApplicationName string                `json:"application"`
 	NumUnits        int                   `json:"num-units"`
 	Placement       []*instance.Placement `json:"placement"`
+	AttachStorage   []string              `json:"attach-storage,omitempty"`
```

Comment:
It won't be, because this is input not output. I do need to bump though, so the client can tell that --attach-storage will or won't be honoured. Which I need to create a card for.